[2025-02-02T16:19:30.483+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T16:19:30.497+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v18.extract_and_load_to_mongodb_station_2 manual__2025-02-02T16:19:29.376213+00:00 [queued]>
[2025-02-02T16:19:30.504+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v18.extract_and_load_to_mongodb_station_2 manual__2025-02-02T16:19:29.376213+00:00 [queued]>
[2025-02-02T16:19:30.504+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-02T16:19:30.516+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_and_load_to_mongodb_station_2> on 2025-02-02 16:19:29.376213+00:00
[2025-02-02T16:19:30.526+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2614) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-02T16:19:30.527+0000] {standard_task_runner.py:72} INFO - Started process 2615 to run task
[2025-02-02T16:19:30.525+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extraction_by_hour_dag_v18', 'extract_and_load_to_mongodb_station_2', 'manual__2025-02-02T16:19:29.376213+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/extract_data_by_hour.py', '--cfg-path', '/tmp/tmpdgzos8kz']
[2025-02-02T16:19:30.530+0000] {standard_task_runner.py:105} INFO - Job 182: Subtask extract_and_load_to_mongodb_station_2
[2025-02-02T16:19:30.568+0000] {task_command.py:467} INFO - Running <TaskInstance: extraction_by_hour_dag_v18.extract_and_load_to_mongodb_station_2 manual__2025-02-02T16:19:29.376213+00:00 [running]> on host a0bc887b90c4
[2025-02-02T16:19:30.635+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extraction_by_hour_dag_v18' AIRFLOW_CTX_TASK_ID='extract_and_load_to_mongodb_station_2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T16:19:29.376213+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T16:19:29.376213+00:00'
[2025-02-02T16:19:30.636+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T16:19:39.946+0000] {logging_mixin.py:190} INFO - Dernier enregistrement pour la station 283181971 (list1) inséré avec succès dans MongoDB.
[2025-02-02T16:19:39.952+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans MongoDB : E11000 duplicate key error collection: air_quality.hourly_data index: _id_ dup key: { _id: ObjectId('679f9b1b9b15065e311dab24') }, full error: {'index': 0, 'code': 11000, 'errmsg': "E11000 duplicate key error collection: air_quality.hourly_data index: _id_ dup key: { _id: ObjectId('679f9b1b9b15065e311dab24') }", 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('679f9b1b9b15065e311dab24')}}
[2025-02-02T16:19:39.952+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T16:19:40.007+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T16:19:40.007+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extraction_by_hour_dag_v18, task_id=extract_and_load_to_mongodb_station_2, run_id=manual__2025-02-02T16:19:29.376213+00:00, execution_date=20250202T161929, start_date=20250202T161930, end_date=20250202T161940
[2025-02-02T16:19:40.048+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T16:19:40.059+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T16:19:40.060+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
